multimedia/youtube-viewer/Portfile

```diff
@@ -0,0 +1,74 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+PortSystem          1.0
+PortGroup           perl5 1.0
+
+perl5.require_variant   yes
+perl5.conflict_variants yes
+perl5.branches          5.26 5.28 5.30
+perl5.default_branch    5.28
+perl5.create_variants   ${perl5.branches}
+
+name                youtube-viewer
+perl5.setup         youtube-viewer 3.9.1
+categories          multimedia net
+platforms           darwin
+maintainers         {@thekevjames thekev.in:macports} openmaintainer
+license             {Artistic-1 GPL}
+description         A very easy interface to YouTube.
+long_description    Lightweight app for searching and streaming videos from YouTube
+homepage            https://trizenx.blogspot.com/2012/03/gtk-youtube-viewer.html
+master_sites        https://github.com/trizen/youtube-viewer/archive/
+distfiles           ${version}.tar.gz
+
+checksums           rmd160  8269ca66628bbf46cc274b5be65ad8d75fdf07aa \
+                    sha256  19a2195c23bcb13b17c2de6bf69b8d4adf74c601812a28823b32b7831caffb97 \
+                    size    257481
+
+variant readline description {Add GNU readline features} {}
+variant perf description {Include performance improvements} {}
+default_variants    +perf +readline
+
+perl5.use_module_build
+perl5.link_binaries_suffix
+
+# TODO: workaround for https://github.com/macports/macports-ports/pull/9004#issuecomment-722054390
+depends_lib \
+    port:perl${perl5.major} \
+    port:p${perl5.major}-module-build \
+
+depends_build-append \
+    port:p${perl5.major}-cpan-meta \
+    port:p${perl5.major}-inc-latest \
+    port:p${perl5.major}-module-build \
+    port:p${perl5.major}-test-harness
+depends_lib-append \
+    port:p${perl5.major}-data-dump \
+    port:p${perl5.major}-encode \
+    port:p${perl5.major}-file-path \
+    port:p${perl5.major}-getopt-long \
+    port:p${perl5.major}-http-message \
+    port:p${perl5.major}-json \
+    port:p${perl5.major}-libwww-perl \
+    port:p${perl5.major}-lwp-protocol-https \
+    port:p${perl5.major}-mime-base64 \
+    port:p${perl5.major}-pathtools \
+    port:p${perl5.major}-scalar-list-utils \
+    port:p${perl5.major}-term-ansicolor \
+    port:p${perl5.major}-term-readline \
+    port:p${perl5.major}-text-parsewords \
+    port:p${perl5.major}-text-tabsxwrap \
+    port:p${perl5.major}-uri
+depends_run-append \
+    port:mpv \
+    port:p${perl5.major}-mozilla-ca
+
+if {[variant_isset readline]} {
+    depends_lib-append \
+        port:p${perl5.major}-term-readline-gnu
+}
+
+if {[variant_isset perf]} {
+    depends_lib-append \
+        port:p${perl5.major}-json-xs \
+        port:p${perl5.major}-lwp-useragent-cached
+}
```

Comment:
please add the extra dependencies for the two variants inside of the variant declaration, that avoids needing to check if they're set here